packages/components/src/unit-control/types.ts

```diff
@@ -71,42 +66,21 @@ export type UnitSelectControlProps = Pick< InputControlProps, 'size' > & {
 	units?: WPUnitControlUnit[];
 };
 
-// TODO: when available, should (partially) extend `NumberControl` props.
 export type UnitControlProps = Omit< UnitSelectControlProps, 'unit' > &
-	Pick<
-		InputControlProps,
-		'hideLabelFromVision' | 'prefix' | '__next36pxDefaultSize'
-	> & {
-		__unstableStateReducer?: StateReducer;
-		__unstableInputWidth?: CSSProperties[ 'width' ];
-		/**
-		 * The children elements.
-		 */
-		children?: ReactNode;
```

Comment:
This looks like it was never supposed to be there? Interesting that it was never spotted before 😅  The benefits of a TypeScript refactor!

Reply:
It was a side effect of a Lodash refactor, just two weeks ago 😆
https://github.com/WordPress/gutenberg/pull/43474/files#diff-6bc8b54c48ea7869398475f3f56904b911610cd33bcdec2be1563707f9657879